test/generated/commands/hincrbyfloat.test.ts

```diff
@@ -20,7 +20,7 @@ test("docs/redis-doc/commands/hincrbyfloat.md example 1", async () => {
     outputs.r3 = await client.hset("mykey", ["field", "5.0e3"]);
     // Error decoding command `HINCRBYFLOAT mykey field 2.0e2`:
 
-    // decoding HINCRBYFLOAT overload 0 (key,field,increment): { name: 'key', schema: { title: 'key', type: 'string' } },{ name: 'field', schema: { title: 'field', type: 'string' } },{ name: 'increment', schema: { title: 'increment', type: 'number' } }
+    // decoding HINCRBYFLOAT overload 0 (key,field,increment): { name: 'key', schema: { title: 'key', type: 'string' }, toString: [Function (anonymous)] },{ name: 'field', schema: { title: 'field', type: 'string' }, toString: [Function (anonymous)] },{ name: 'increment', schema: { title: 'increment', type: 'number' }, toString: [Function (anonymous)] }
```

Comment:
Not sure why these appeared, are you using node 14 maybe?
```suggestion
    // decoding HINCRBYFLOAT overload 0 (key,field,increment): { name: 'key', schema: { title: 'key', type: 'string' } },{ name: 'field', schema: { title: 'field', type: 'string' } },{ name: 'increment', schema: { title: 'increment', type: 'number' } }
```

Reply:
I'm on 15.9.0